lib/screens/b_quiz/quiz_repository.dart

```diff
@@ -0,0 +1,219 @@
+              "https://source.unsplash.com/random/?food/"
+            ]
+          },
+          {
+            "question": "How far is the moon from Earth?",
+            "answers": [
+              "7,918 miles (12,742 km)",
+              "86,881 miles (139,822 km)",
+              "238,400 miles (384,400 km)",
+              "35,980,000 miles (57,910,000 km)"
+            ],
+            "correctIndex": 2,
+            "images": [
+              "https://source.unsplash.com/random/?Cryptocurrency/",
+              "https://source.unsplash.com/random/?business/",
+              "https://source.unsplash.com/random/?food/"
+            ]
+          },
+          {
+            "question": "What is 65 times 52?",
+            "answers": ["117", "3120", "3380", "3520"],
+            "correctIndex": 2,
+            "images": [
+              "https://source.unsplash.com/random/?Cryptocurrency/",
+              "https://source.unsplash.com/random/?business/",
+              "https://source.unsplash.com/random/?food/"
+            ]
+          },
+          {
+            "question": "How tall is Mount Everest?",
+            "answers": [
+              "6,683 ft (2,037 m)",
+              "7,918 ft (2,413 m)",
+              "19,341 ft (5,895 m)",
+              "29,029 ft (8,847 m)"
+            ],
+            "correctIndex": 3,
+            "images": [
+              "https://source.unsplash.com/random/?Cryptocurrency/",
+              "https://source.unsplash.com/random/?business/",
+              "https://source.unsplash.com/random/?food/"
+            ]
+          },
+          {
+            "question": "When did The Avengers come out?",
+            "answers": [
+              "May 2, 2008",
+              "May 4, 2012",
+              "May 3, 2013",
+              "April 4, 2014"
+            ],
+            "correctIndex": 1,
+            "images": [
+              "https://source.unsplash.com/random/?Cryptocurrency/",
+              "https://source.unsplash.com/random/?business/",
+              "https://source.unsplash.com/random/?food/"
+            ]
+          },
+          {
+            "question": "What is 48,879 in hexidecimal?",
+            "answers": ["0x18C1", "0xBEEF", "0xDEAD", "0x12D591"],
+            "correctIndex": 1,
+            "images": [
+              "https://source.unsplash.com/random/?Cryptocurrency/",
+              "https://source.unsplash.com/random/?business/",
+              "https://source.unsplash.com/random/?food/"
+            ]
+          }
+        ]
+      };
+      return (response["questions"] as List)
+          .map((e) => Questions.fromJson(e))
+          .toList();
+    }
+  }
+}
+
+class APIQuizRepository extends QuizRepository {
+  final String classTag = "APIgetQuizRepository";
+  @override
+  Future<List<Questions>> getAllQuizes() async {
+    final String tag = classTag + "getAllQuizs()";
+    http.Response response;
+    Uri url = Uri.parse("http://43.205.53.122/bquiz/Quizboard/");
```

Comment:
change it to base url Use the string constants already present there

Reply:
@TreavVasu sir issue has been resolved.